src/Symfony/Bundle/FrameworkBundle/Resources/config/form.php

```diff
@@ -123,6 +124,10 @@
             ->args([service('form.type_extension.form.request_handler')])
             ->tag('form.type_extension')
 
+        ->set('form.type_extension.form.flow.session_data_storage', FormFlowTypeSessionDataStorageExtension::class)
```

Comment:
I like that this storage (default to session) is internal since day one, will help a lot to avoid ppl mistake I think

Reply:
Yep, new data storages can also be added later.